inference-engine/src/offline_transformations/src/pot_transformations.cpp

```diff
@@ -14,11 +14,8 @@ NGRAPH_RTTI_DEFINITION(ngraph::pass::POTTransformations, "POTTransformations", 0
 
 bool ngraph::pass::POTTransformations::run_on_function(std::shared_ptr<ngraph::Function> f) {
     ngraph::pass::Manager manager(get_pass_config());
-    if (m_device == "CPU") {
-        // TODO: register CPU passes
-        // manager.register_pass<ngraph::pass::LSTMCellDecomposition>();
-    } else {
-        throw ngraph_error("Device name is unsupported");
+    if (m_device == "GNA") {
+        manager.register_pass<ngraph::pass::LSTMCellDecomposition>();
```

Comment:
After this one transformation in POT we will have a potential problem with unrolled topologies due to missing `Identity`. We need to rewrite `InsertIdentityToLSTMCellPass` transformation to ngraph without searching LSTMCell (task 52095).

It is not applicable to POT, it should be fixed in GNA plugin. Just to notify, that such topologies can have all zeros on output.